src/book.json

```diff
@@ -17,27 +17,29 @@
     "geoServerPassword": "geoserver",
     "geoServerVersion": "2.18.1",
     "osGeoLiveVersion": "14.0",
-    "authors": [{
-      "name": "Nils Bühner",
-      "mail": "buehner@terrestris.de"
-    }, {
-      "name": "André Henn",
-      "mail": "henn@terrestris.de"
-    }, {
-      "name": "Daniel Koch",
-      "mail": "koch@terrestris.de"
-    }, {
-      "name": "Fabian Schmidt",
-      "mail": "fschmidt@terrestris.de"
-    }]
+    "authors": [
+      {
+        "name": "Nils Bühner",
+        "mail": "buehner@terrestris.de"
+      },
+      {
+        "name": "André Henn",
+        "mail": "henn@terrestris.de"
+      },
+      {
+        "name": "Daniel Koch",
+        "mail": "koch@terrestris.de"
+      },
+      {
+        "name": "Fabian Schmidt",
+        "mail": "fschmidt@terrestris.de"
+      }
+    ]
   },
   "plugins": [
-    "advanced-emoji",
     "edit-link",
     "expandable-chapters",
-    "image-captions",
-    "richquotes",
```

Comment:
Please also delete the plugin configurations for the removed plugins.

Reply:
fixed with b4eb23c18a4959b415270cd3798e96c74e719e1d